test/runnable/selectiveUnittests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+unittest_output="unittesting printingUnittest"
+
+$DMD -m${MODEL} -I=${EXTRA_FILES} -unittest ${EXTRA_FILES}/printingUnittest.d -run ${EXTRA_FILES}/selectiveUnittests.d | grep -q "${unittest_output}"
```

Comment:
The error code for this isn't 0, but it won't cause the whole script to fail.

Reply:
ah...I see you're not familiar with the latest work on DMD's Bash tests...see `test/tools/sh_do_test.sh` :)